components/messenger/transports.rst

```diff
@@ -0,0 +1,225 @@
+.. index::
+    single: Messenger; Transports
+
+Transports
+==========
+
+To send and receive messages via message brokers, the Messenger component has
+"transports". Transports are responsible for routing messages to and from
+the message brokers.
+
+Every transport is configurable using a DSN. This DSN allows you to chose the
+transport layer as well as configure it.
+
+AMQP
+----
+
+The most famous message broker protocol is probably AMQP, which is most
+commonly implemented with RabbitMQ. The Messenger component has built-in
+support for AMQP.
+
+How it works?
+~~~~~~~~~~~~~
+
+A DSN protocol that uses ``amqp`` is recognized and sent to the built-in AMQP transport.
```

Comment:
DSN isn’t a protocol isn’t it?

Reply:
@OskarStark the idea is to remove only `protocol` word? or change it to something else?

Reply:
maybe:
```suggestion
A DSN that starts with ``amqp://`` is recognized and used to create an instance of the built-in AMQP transport.
```

Reply:
Good suggestion, thank you @greg0ire

Reply:
thanks!